### PR TITLE
Create README for discord-bot

### PR DESCRIPTION
**Pull request by arionrefat**

### CODE REVIEW

File: packages/discord-bot/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3. `discord.js`: Powerful Node.js module to interact with the Discord API
+
+## Folder organization
+
+- `dist`: Hidden folder created by typescript to emit all the transpiled js server code
+- `src`: Source code
+  - `commands`: This is where all the commands are written
+  - `config.ts`: Exports environment variables to be consumed with guaranteed type safety and defaults
+  - `client.ts`: this is what we use to communicate with discord client
+  - `types`: Types exported from `@packages/server`
```
Comment (Devorein) [inline packages/discord-bot/README.md:22]: There is nothing named `@packages/server`

File: packages/discord-bot/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Environment variables
+
+**NOTE** This must be named `.env`
```
Comment (Devorein) [inline packages/discord-bot/README.md:26]: These are server environment variables, not for discord. The ones for discord-bot are
1. `DISCORD_CLIENT_ID`
2. `DISCORD_BOT_TOKEN`
3. `DISCORD_GUILD_ID`

Please explain what each of them do and how to obtain them (via link to an article)

File: packages/discord-bot/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# `@reinforz/discord-bot`
+
+## Setup
+
+1. Create a `.env` file with the keys mentioned [here](#environment-variables-development)
```
Comment (Devorein) [inline packages/discord-bot/README.md:5]: Clicking **here** doesn't navigate to the correct place. Please fix this

File: packages/discord-bot/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- `SERVER_PORT`: Port where the server will listen for incoming requests
+- `DB_URL`: Connection string for database (includes username, password, host, port and database name)
+- `CLIENT_URL`: Url where the next.js server (frontend) is hosted/running
+- `PASSWORD_SALT`: Random string added to password before hashing to strengthen it
+- `JWT_SECRET`: Random string added before signing a JWT
+- `JWT_ALGORITHM`: Algorithm used to sign jwt token
+- `NODE_ENV`: Current node environment (`development`, `staging`, `testing` & `production`)
+- `SUDO_API_KEY`: This is where discord sudo api key is here
+
+**Note** See `.env.test` in the server package directory to see an example environment variables file.
```
Comment (Devorein) [inline packages/discord-bot/README.md:38]: No need to mention this as server and discord-bot environment variables are completely different

File: packages/discord-bot/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# `@reinforz/discord-bot`
+
+## Setup
+
+1. Create a `.env` file with the keys mentioned [here](#environment-variables-development)
+2. Run `npm run dev` to start the development server and the tsc compiler(in watch mode) simultaneously.
+3. After running the above commands the console should show 'Ready, Then you can send command `!users` in Reinforz discord server.
```
Comment (Devorein) [inline packages/discord-bot/README.md:7]: Write a section for the supported commands. Like so
1. `!users`: Show the number to public, private and total users
2. ....

**Review by Devorein — CHANGES_REQUESTED** [review]

File: packages/discord-bot/README.md (hunk 1 above)
Comment (arionrefat) [reply packages/discord-bot/README.md:22]: @reinforz/server?

Comment (Devorein) [reply packages/discord-bot/README.md:22]: Yep, we don't have it linked with `@reinforz/server` yet but that's the plan. So just replace this with `@reinforz/server`

**Review by Devorein — APPROVED** [review]

**Pull request merged**